site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -196,6 +196,8 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_type!:
 :generator_display_name!:
 +
+[field]#tenant.externalIdentifierConfiguration.samlv2AuthNRequestIdTimeToLiveInSeconds# [type]#[Integer]# [optional]#Optional# [default]#300# [since]#Available since 1.20.0#::
+The time in seconds that a SAML AuthN request will be eligible for use to authenticate with FusionAuth.
```

Comment:
This should be alphabetized, so it should be above `tenant.externalIdentifierConfiguration.setupPasswordIdGenerator`